CONFIGURATION.md

```diff
@@ -89,6 +89,12 @@ modules:
   # Probe fails if SSL is not present.
   [ fail_if_not_ssl: <boolean> | default = false ]
 
+  # Probe fails if response body JSON matches CEL:
+  fail_if_body_matches_cel: <cel expression, root field is called body>
```

Comment:
this should be `fail_if_body_json_matches_cel`, right?

Reply:
Correct